test/images/shellcheck/Dockerfile

```diff
@@ -18,10 +18,12 @@ WORKDIR /
 RUN mkdir /workspace && chmod 777 /workspace && chown 65532:65532 /workspace
 ENV HOME /tmp/home
 RUN mkdir $HOME && chmod 777 $HOME && chown 65532:65532 $HOME
-RUN microdnf install tar xz findutils
+RUN microdnf install -y tar-1.30 xz-5.2.4 findutils-4.6.0 && microdnf clean all
+SHELL ["/bin/sh", "-eo", "pipefail", "-c"]
 RUN SHELLCHECK_VERSION=0.7.1 && \
```

Comment:
Looks like the `v` should be part of the version (i.e. `v0.7.1`)

Reply:
![image](https://user-images.githubusercontent.com/74113200/187136693-35011d01-b749-4d02-8cdd-a50c9009664c.png)
You are right. But we are appending `v` in the name string wherever required. So we may append `v` here and remove it everywhere else.

Reply:
same as above, fixed